repository_service_tuf_worker/models/targets/crud.py

```diff
@@ -119,19 +119,20 @@ def update(
     return target
 
 
-def update_to_published(
-    db: Session, target: models.RSTUFTargets
-) -> models.RSTUFTargets:
+def update_to_published(db: Session, paths: List[str]) -> None:
     """
     Update Target to `published` to `True`.
     """
-    target.published = True
-    target.last_update = datetime.now()
-    db.add(target)
-    db.commit()
-    db.refresh(target)
 
-    return target
+    db.query(models.RSTUFTargets).filter(
+        models.RSTUFTargets.path.in_(paths)
+    ).update(
+        {
+            models.RSTUFTargets.published: True,
+            models.RSTUFTargets.last_update: datetime.now(),
+        }
+    )
+    db.commit()
```

Comment:
I see you also did `db.refresh(target)` before.
This is not required anymore because `db.update()` does that job right?

Reply:
The `db.refresh()` is required when you will return the object, not to return it with the initial value.
We no long return the object because we don't give it to the function, only the paths.